src/app/design/styled-button/styled-button.component.scss

```diff
@@ -0,0 +1,71 @@
+@import '../theme-colors';
```

Comment:
Use global scss (see `_variables.scss`)

Reply:
I used the theme colors file to be able to add more themes later (if wanted).

Reply:
Its also easier to edit the theme data if its in a specific file.